README.md

```diff
@@ -68,134 +62,58 @@ then as soon as your PR is merged into master, it will be merged into the curren
 
     - Add a new FacetItem at the top of the file containing the previous release's version number. In
-    this case, this would be "V1.3".
+    this case, this would be "V1.6".
 
     - Upload the updated CSE context file to the site.
 
     - In the Setup section, add a new site that covers the previous release's archive directory. In this
-    case, the site URL would be archive.istio.io/v1.3/*. Set the label of this site to the name of the
-    facet item created above (V1.3 in this case).
-
-1. In the **previous release's** branch (in this case `release-1.3`), edit the file `data/args.yml`. Set the
-`archive` field to true and the `archive_date` field to the current date, and the `archive_search_refinement`
-to the previous release version (in this case `V1.3`).
-
-1. In the **previous release's** branch (in this case `release-1.3`), edit the file `config.toml`. Set the
-`disableAliases` field to `false`.
-
-1. Commit the previous edits to your local git repo and push the **previous release's** branch to GitHub.
-
-1. In the **archive** branch, rebase the branch to have all changes from the current release. In this case,
-all changes from the `release-1.4` branch.
-
-1. Commit the previous edits to your local git repo and push the **archive** branch to GitHub.
-
-1. Wait a while (~20 minutes) and browse archive.istio.io to make sure everything looks good.
-
-##### Updating preliminary.istio.io
-
-1. In the **master** branch, edit the file `data/args.yml`. Set the `version` and `full_version` fields to have the version
-of the next Istio release, and `previous_version` to be the version of the previous release. In this case, you would set the fields to
-"1.5", "1.5.0", and "1.4" respectively.
-
-1. In the **master** branch, edit the file `data/args.yml`. Set the
-`source_branch_name` and `doc_branch_name` fields to `master`.
-
-1. In the **master** branch, edit the file `Makefile.core.mk`. Set the variable `SOURCE_BRANCH_NAKE` to
-`master`.
-
-1. Run `make update_all` in order to retrieve the latest material from the source repositories.
-
-1. Commit the previous edits to your local git repo and push the **master** branch to GitHub.
-
-1. Wait a while (~5 minutes) and browse preliminary.istio.io to make sure everything looks good.
+    case, the site URL would be archive.istio.io/v1.6/*. Set the label of this site to the name of the
+    facet item created above (V1.6 in this case).
 
 ### Creating a patch release
 
 A few days before the patch release, the release managers should notify the Doc WG that the release
 is built and is starting it's long running qualification test. At this time, move the doc automation
-tests to use the new release to verify automated doc testing passes. To move to a new release
-(make sure you are in the patch's release branch):
-
-1. `go get istio.io/istio@1.X.Y`
+tests to use the new release to verify automated doc testing passes. 
 
-1. `go mod tidy`
+To create a new patch release, you need to
```

Comment:
How much of the above process applies to patch releases as well? None?  Either way, I think we should reorganize the content to be something like:

```
Creating a Major/Minor release
...
Creating a Patch release
...
```

WDYT?

Reply:
Sounds good. I'll change it right away.